x-pack/plugins/apm/public/tutorial/config_agent/commands/rails.ts

```diff
@@ -5,24 +5,48 @@
  * 2.0.
  */
 
+import { i18n } from '@kbn/i18n';
+import {
+  serviceNameHint,
+  secretTokenHint,
+  serverUrlHint,
+  serviceEnvironmentHint,
+} from './shared_hints';
+
 export const railsVariables = {
   apmServiceName: 'service_name',
   secretToken: 'secret_token',
   apmServerUrl: 'server_url',
   apmEnvironment: 'environment',
 };
 
+export const railsHighlightLang = 'rb';
+
+export const railsLineNumbers = {
+  start: 1,
+  highlight: '3, 5, 7, 9',
+  annotations: {
+    3:
+      serviceNameHint +
+      ' ' +
+      i18n.translate(
+        'xpack.apm.tutorial.railsClient.createConfig.commands.defaultServiceName',
+        {
+          defaultMessage: 'Defaults to the name of your Rails app.',
+        }
+      ),
```

Comment:
Suggestion for these kind of compound hints:
```suggestion
const railsNameHint = i18n.translate(
  'xpack.apm.tutorial.railsClient.createConfig.commands.defaultServiceName',
  { defaultMessage: 'Defaults to the name of your Rails app.' }
);
// ...
  annotations: {
    3: `${serviceNameHint} ${railsNameHint}`,
```

Reply:
Neat! I'll update to this format. Thanks 😄

Reply:
Sorry that took so long! Updated.